app/lib/frontend/templates/views/pkg/package_list.dart

```diff
@@ -146,15 +150,25 @@ d.Node _packageItem(
 }
 
 d.Node _item({
+  required List<ProcessedScreenshot>? screenshots,
   required String url,
   required String name,
+  required String? version,
   required DateTime? newTimestamp,
   required d.Node? labeledScoresNode,
   required String description,
   required d.Node metadataNode,
   required d.Node? tagsNode,
   required List<_ApiPageUrl>? apiPages,
 }) {
+  final bool hasScreenshots = screenshots != null && screenshots.isNotEmpty;
```

Comment:
maybe add `&& version != null` just in case, as the `getImageURL` uses `version!`

Reply:
done